frontend/src/app/artshow/artshow.component.html

```diff
@@ -0,0 +1,36 @@
+<app-breadcrumb></app-breadcrumb>
+<div class="row">
+    <div class="col-12">
+        <h1>Welcome to the Art Show!</h1>
+    </div>
+    <div class="col-md-4 col-12">
+        <app-callout-box>
+            <div class="callout-box-title">
+                Art Show Tours
+            </div>
+            <div class="callout-box-body">
+                <p>Guest of Honor Tour: Day/Time, link</p>
+                <p>Descriptive Tour: Day/Time, link</p>
+            </div>
+            <div class="callout-box-title">
+                Chat with Artists!
+            </div>
+            <div class="callout-box-body">
+                <p>Schedule: link</p>
+                <p>Zoom: link</p>
+            </div>
+        </app-callout-box>
+    </div>
+    <div class="col-md-8 col-12">
+        <div class="row">
+            <div class="col-12">
+                <p>Select the image above an artist's name to view their Art Show gallery.
```

Comment:
Maybe just "click on an artist"? You can click on the picture or the name underneath right?

Reply:
Yeah. I'll change it. Sounds better.